pyscript.core/tests/ws/README.md

```diff
@@ -0,0 +1,34 @@
+# PyScript Test Suite
+
+The test suite is a bit of a mess. We need to fix it.
```

Comment:
Isn't the purpose of this MR to fix this status? Just wondering if this file needs any update or if we should just remove (and I'll follow up with more cleanup around JS, if needed).